src/main/resources/CodyBundle.properties

```diff
@@ -183,5 +183,9 @@ gotit.autocomplete.message=This is how Cody displays autocomplete suggestions.<b
 
 # Other Actions
 action.cody.restartAgent.text=Restart Cody Agent
-ignore.action-in-ignored-file.detail=Cody has ignored this file because of your Sourcegraph admin policy
-ignore.action-in-ignored-file.learn-more-cta=Learn more
+ignore.action-in-ignored-file.detail=This file has been ignored by an admin. Autocomplete, commands, and other Cody features are disabled.
```

Comment:
"Ignored by an admin" sounds a bit ambiguous. How about "excluded" or "put on the ignore list"?

Reply:
These strings come from our UX designers @toolmantim and @danielmarquespt (although I took the liberty of replacing "repo" with "file" in some instances... "file" will be more durable if planned changes to Ignore arrive and we don't loop back to update these.)

Reply:
The "By admin" has the objective of letting the user know that there is nothing they can do, or is not a setting.

Saying "excluded" or on an "ignore list" might make the user think its a local setting or and make then look for it thinking they can actually change it.

However the actual wording can be tweaked